### PR TITLE
ability to pass object to be tribble_paste()d

### PR DESCRIPTION
**Pull request by jonocarroll**
as per https://twitter.com/MilesMcBain/status/820473995787051010

and relevant to https://twitter.com/hadleywickham/status/837384888856674306

### CODE REVIEW

File: R/tribble_paste.R [hunk 1]
```diff
@@ -5,7 +5,13 @@ globalVariables(".rs.readUiPref", "datapasta") #ignore this function in R CMD ch
...
 #' @export
 #'
-tribble_paste <- function(){
+tribble_paste <- function(x) {
+
+  # if data is provided, try to use that to generate the tribble_paste()
+  if(!missing(x)) {
+    clipr::write_clip(x)
+  }
+
```
Comment (MilesMcBain) [inline R/tribble_paste.R]: I think we should just be able to do clipboard_table <- x. Might be safest to do as.data.frame(x) or test if x is the right class.

Comment (MilesMcBain) [inline R/tribble_paste.R]: This way it will work on remote systems! ;)

Comment (jonocarroll) [inline R/tribble_paste.R]: Cool. I just took your tweet example and ran with it. `clipboard_table <- x` was my first guess but I wasn't sure what form `x` was expected to be in further down the line. I'll have a go at fixing it up tonight, I was rushed to get even that much written.

Comment (MilesMcBain) [inline R/tribble_paste.R]: Cool I had a bash earlier in the tibble_out branch. This has revealed a complication in the way I was handling NAs. For example try tribble_paste(head(airquality)) to see a bunch of NAs converted to NaNs.

**Comment by MilesMcBain** [comment]
Could use a few more tests but I am going to do a bit of a refactor next.

**Pull request closed without merging**